lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/doi/DoiId.json

```diff
@@ -0,0 +1,26 @@
+{
+	"description": "JSON schema for all fields associated with a DOI and its metadata.",
+	"type":"object",
```

Comment:
convert to interface that has all of our fields (and have DoiV2 not create any new fields)